proto/da/da.proto

```
@@ -130,3 +130,19 @@ message ValidateRequest {
 message ValidateResponse {
 	repeated bool results = 1;
 }
+
+enum ErrorCode {
+	Unknown = 0;
```

buf

Enum value name "Unknown" should be prefixed with "ERROR_CODE_".

(ENUM_VALUE_PREFIX)